security/vertx-jwt/src/main/java/io/quarkus/ts/security/vertx/config/AuthNConfig.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.ts.security.vertx.config;
+
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithName;
+
+@ConfigMapping(prefix = "authN")
+public interface AuthNConfig {
+    String alg();
+
+    String secret();
+
+    @WithName("tokenLiveSpanMin")
```

Comment:
As far I know, this is not working. This was fixed in beefy as part of this commit: https://github.com/quarkus-qe/beefy-scenarios/commit/cc3fe7348b9ae582890037ae27c83ce12ce9891b